server/src/domain/search/search.service.ts

```diff
@@ -103,6 +103,34 @@ export class SearchService {
     return this.mapResponse(items, hasNextPage ? (page + 1).toString() : null);
   }
 
+  async searchHybrid(auth: AuthDto, dto: HybridSearchDto): Promise<SearchResponseDto> {
+    if (dto.context) {
+      await this.configCore.requireFeature(FeatureFlag.SMART_SEARCH);
+    } else {
+      await this.configCore.requireFeature(FeatureFlag.SEARCH);
+    }
```

Comment:
Hybrid search is a specific kind of CLIP search that also does full-text search on text fields (not just filtering) and merges the ranking from the two. This endpoint would always require smart search.